packages/desktop/views/onboarding/views/create-profile/views/ChooseCreateProfileFlowView.svelte

```diff
@@ -27,8 +27,10 @@
 
     async function onProfileTypeClick(createProfileType: CreateProfileType): Promise<void> {
         isBusy = { ...isBusy, [createProfileType]: true }
-        const type = createProfileType === CreateProfileType.Ledger ? ProfileType.Ledger : ProfileType.Software
-        updateOnboardingProfile({ createProfileType, type })
+        const isLedgerProfile = createProfileType === CreateProfileType.Ledger
```

Comment:
ideally, you should also update `buildInitialOnboardingProfile` in `packages/shared/lib/contexts/onboarding/helpers/buildInitialOnboardingProfile.ts` so it doesnt initialize the stronghold version from the very begining